src/vorta/views/schedule_tab.py

```diff
@@ -149,6 +155,12 @@ def populate_from_profile(self):
         self.meteredNetworksCheckBox.setChecked(
             QtCore.Qt.Unchecked if profile.dont_run_on_metered_networks else QtCore.Qt.Checked
         )
+        self.allowNewNetworksCheckBox.setChecked(
+            QtCore.Qt.Checked if profile.allow_new_networks else QtCore.Qt.Unchecked
+        )
```

Comment:
```suggestion
        self.allowNewNetworksCheckBox.setChecked(
            profile.allow_new_networks
        )
```

`setChecked` expects a boolean. Therefore the if isn't needed.